src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -147,10 +147,6 @@ public void load(CoreExtension.Context context) {
                             "Repository Slug see for example https://docs.atlassian.com/bitbucket-server/rest/latest/bitbucket-rest.html")
                             .type(PropertyType.STRING).build(),
 
-                    PropertyDefinition.builder(BitbucketServerPullRequestDecorator.PULL_REQUEST_BITBUCKET_USER_SLUG).category(PULL_REQUEST_CATEGORY_LABEL).subCategory(BITBUCKET_INTEGRATION_SUBCATEGORY_LABEL)
```

Comment:
I deleted this as we don't use it anymore and it just confuses the users

Reply:
👍